Assets/Scripts/Models/Character.cs

```diff
@@ -67,13 +67,13 @@ public float Y
         }
     }
     Need[] needs;
-    
+
     /// <summary>
     /// The tile the Character is considered to still be standing in.
     /// </summary>
     public Tile CurrTile
     {
-        get 
+        get
         {
             return _currTile;
```

Comment:
@Grenkin1988 Last time I checked CONTRIBUTING.md, that was exactly how it should be done.